misc/metrics/src/swarm.rs

```diff
@@ -123,6 +124,13 @@ impl Metrics {
             Box::new(connections_closed.clone()),
         );
 
+        let connection_duration = Family::new_with_constructor(create_connection_duration_histogram as fn()->Histogram);
+        sub_registry.register(
+            "connection_duration",
+            "Histogram of milliseconds it has taken to move a connection from pending to established",
```

Comment:
```suggestion
            "Time it took to establish a connection",
```

Reply:
Would be easier for me to understand.

Reply:
The problem I was trying to dance around - the 'time to establish connection' starts before the dial. And depending on direction that may be a different machine, and outside the measurement.